packages/components/src/Button/_docs/Button.stickersheet.stories.tsx

```diff
@@ -0,0 +1,241 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { CheckIcon } from "~components/SVG/icons/CheckIcon"
+import { StickerSheet } from "../../../../../storybook/components/StickerSheet"
+import { Button, ButtonProps } from "../index"
+
+export default {
+  title: "KAIO/Button",
```

Comment:
Ohhh smart. Should do this with FilterMS too 🤔

Reply:
Yeah, needed a way to make sure my changes didn't outright break the Button haha